test/e2e/serial/tests/configuration.go

```diff
@@ -400,8 +409,9 @@ var _ = Describe("[serial][disruptive][slow] numaresources configuration managem
 			err = kubeletconfig.KubeletConfToMCKubeletConf(kcObj, targetedKC)
 			Expect(err).ToNot(HaveOccurred())
 
-			err = fxt.Client.Update(context.TODO(), targetedKC)
-			Expect(err).ToNot(HaveOccurred())
+			Eventually(func() error {
+				return fxt.Client.Update(context.TODO(), targetedKC)
```

Comment:
wrapping the update like this can be not enough: we need to make sure we get a new copy of the object we want to update, we apply our changes on top and we sent an update at the end, all wrapped in a retry loop